.gitlab-ci.yml

```diff
@@ -84,67 +84,16 @@ trigger_pipeline:
       fi
 
 
-# Build jobs
-# Job with example runs.
-# cuda 9.2 and friends
-build/cuda92/nompi/gcc/all/release/shared:
```

Comment:
I will suggest to bump the version after release.

Reply:
We still have compatibility, but to make space for the new CUDA versions I remove the old ones.

Reply:
After removing them, it's hard to track whether the new functionalities are available in the earliest version or not.
Keeping one job for 9.2 to ensure we still testing our minimal requirement.
same for dpcpp 2022.1

Reply:
I agree with CUDA, but Intel's ecosystem is still so much of a moving target that I would rather stay at head